datamap/mv/core/src/main/scala/org/apache/carbondata/mv/rewrite/Navigator.scala

```diff
@@ -206,4 +312,59 @@ private[mv] class Navigator(catalog: SummaryDatasetCatalog, session: MVSession)
     }
     true
   }
+
+  /**
+   * Replace the granularity in the plan
```

Comment:
please explain more, what is replaced with what?

Reply:
Added description. Please check